OxfordBodleian/Juel-Jensen/BDLaethd19.xml

```diff
@@ -224,9 +224,196 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             </handNote>
                         </handDesc>
 
-                       <!-- <decoDesc>
+                        <decoDesc>
+                            <summary>
+                              This psalter contains iconic portraits of figures from the Old and New Testaments.
+                              It is among the most extensively illustrated examples from the mid-fifteenth century.
+                              Most miniatures take up a full page, but some ½ or less (cf. fols 4v, 146v).
+                              In all likelihood, this manuscript was scribed and decorated by Gabra Krǝstos,
```

Comment:
mark up with PRS13458GabraK?

Reply:
Should I mark up a hypothetical though?

Reply:
Since you state the probability very clearly verbally, I think it is appropriate here.

Reply:
Thanks, done!